sw/host/opentitanlib/src/transport/cw310/mod.rs

```diff
@@ -231,6 +230,19 @@ impl Transport for CW310 {
             usb.pll_out_enable(2, false)?;
             usb.pll_write_defaults()?;
             Ok(None)
+        } else if let Some(reset_and_clear) = action.downcast_ref::<ResetAndClearUartRx>() {
+            // Assert reset signal (active low) and sleep.
```

Comment:
nit: This comment is incomplete and maybe not necessary, as the code reflects the information in it pretty well